### DISCUSSION TITLE
No sound and dropouts or crash when one plugin is not available/initialized in a chain

### DISCUSSION THREAD

**Discussion by Henri75** [apohl79/audiogridder#821] [Help]
I noticed a small bu anoying issue : 

In a chain of plugin in AG, if one plugin is not available in a chain (for example if you change the server), or if there is any problem with the plugin (crash, initialization issue with AG etc ...) the whole chain is not working and you have no sound, plus some dropouts due to a connection/disconnection loop.

It seems in some case that the sanbox is crashing or terminating itself if there is any issue with one plugin.

It should bypass or disabled the concerned plugin so that the chain is working and the project is usable.

**Comment by yoshimodular** [comment]
In Nuendo/Mac it crashes when a plugin needs a lot of time to load. It happens sometimes with Waves and with Softube.

**Comment by Henri75** [comment]
Are you sure it is related to loading time ?
It could be an issue in loading the plugin or initializing the
connection/szndbox.
Did you check the log ?

On mac softube and waves Load fast, and a lot faster than windows versions
